src/rdiff_backup/rpath.py

```diff
@@ -982,6 +982,9 @@ def new_index_empty(self, index):
         return self.__class__(self.conn, self.base, index, {'type': None})
 
     def get_temp_rpath(self, sibling=False):
+        import tempfile
```

Comment:
No import in functions please, always import at the top of the file, in alphabetical order is better (I know, the current code doesn't do this, but it should, feel free to fix at the same time).
Reference: https://www.python.org/dev/peps/pep-0008/#id23

Reply:
Addressed by https://github.com/rdiff-backup/rdiff-backup/pull/545/commits/fab0481dd833a1f67fab980f671d2e9861c8b747.